### PR TITLE
changed login and sign up textfields and buttons design

### PR DESCRIPTION
**Pull request by defraimo**
- added colour to Colors.swift -> dizzyBlue
- changed layout in Login page: buttons design, fonts and colours
- modified loginTextfield design in TextFieldBuilder

### CODE REVIEW

File: Dizzy/Screens/SignIn/SignInDetailsView.swift [hunk 1]
```diff
@@ -59,7 +59,7 @@ final class SignInDetailsView: UIView {
     private func layoutStackView() {
         self.addSubviews([self.stackView])
         self.stackView.snp.makeConstraints { stackView in
-            stackView.top.equalTo(titleLabel.snp.bottom).offset(Metrics.doublePadding)
+            stackView.top.equalTo(titleLabel.snp.bottom).offset(Metrics.doublePadding * 2)
```
Comment (menesh2) [inline Dizzy/Screens/SignIn/SignInDetailsView.swift]: don't use `*2`, you can create a constant like `stackViewTopPadding = ..` or create in Metrics new variable and use it

File: Dizzy/Screens/SignIn/SignInDetailsView.swift [hunk 2]
```diff
@@ -81,7 +81,7 @@ final class SignInDetailsView: UIView {
     private func layoutSignInButton() {
         stackView.addArrangedSubview(signInButton)
         signInButton.snp.makeConstraints { (signInButton ) in
-            signInButton.width.equalToSuperview().multipliedBy(buttonsWidthPrecentage)
+            signInButton.width.equalToSuperview().multipliedBy(buttonsWidthPrecentage * 0.6)
```
Comment (menesh2) [inline Dizzy/Screens/SignIn/SignInDetailsView.swift]: same as my note in `SingleDetailsView` dont use `*`, create constant called `signInButtonWidthPrecentage` or similar name and use it

File: Dizzy/Entities/Colors.swift [hunk 3]
```diff
@@ -30,4 +30,7 @@ extension UIColor {
     static var primeryPurple: UIColor {
         return UIColor(red: 122, green: 105, blue: 177)
     }
+    static var dizzyBlue: UIColor {
```
Comment (menesh2) [inline Dizzy/Entities/Colors.swift:33]: nice!

File: Dizzy/Screens/SignUp/SignUpDetailsView.swift [hunk 4]
```diff
@@ -64,7 +64,7 @@ final class SignUpDetailsView: UIView {
     private func layoutStackView() {
         self.addSubviews([self.stackView])
         self.stackView.snp.makeConstraints { stackView in
-            stackView.top.equalTo(titleLabel.snp.bottom).offset(Metrics.doublePadding)
+            stackView.top.equalTo(titleLabel.snp.bottom).offset(Metrics.doublePadding * 2)
```
Comment (menesh2) [inline Dizzy/Screens/SignUp/SignUpDetailsView.swift]: same as my other comments for the `*`, dont use it

File: Dizzy/Screens/SignUp/SignUpDetailsView.swift [hunk 5]
```diff
@@ -100,7 +100,7 @@ final class SignUpDetailsView: UIView {
     private func layoutSignupButton() {
         stackView.addArrangedSubview(signUpButton)
         signUpButton.snp.makeConstraints { (signUpButton ) in
-            signUpButton.width.equalToSuperview().multipliedBy(buttonsWidthPrecentage)
+            signUpButton.width.equalToSuperview().multipliedBy(buttonsWidthPrecentage * 0.6)
```
Comment (menesh2) [inline Dizzy/Screens/SignUp/SignUpDetailsView.swift]: same as my other comments for the *, dont use it

File: Dizzy/UI Components/TextFieldBuilder.swift [hunk 6]
```diff
@@ -56,12 +56,21 @@ extension UITextField {
...
-        return self
+          backgroundColor = UIColor.lightGray.withAlphaComponent(0.2)
+          borderStyle = .none
+          layer.cornerRadius = 12
+          font = Fonts.h10()
+          textAlignment = .left
+          textColor = .black
+          attributedPlaceholder = NSAttributedString(string: placeholder ,attributes: [NSAttributedString.Key.foregroundColor: UIColor.gray])
+              
+          let leftPaddingView = UIView(frame: CGRect(x: 0, y: 0, width: Metrics.oneAndHalfPadding, height: 0))
```
Comment (menesh2) [inline Dizzy/UI Components/TextFieldBuilder.swift:67]: you dint need to create UIView in order to get space, look for examples in this project

**Review by menesh2 — CHANGES_REQUESTED** [review]

File: Dizzy/UI Components/TextFieldBuilder.swift (hunk 6 above)
Comment (defraimo) [reply Dizzy/UI Components/TextFieldBuilder.swift:67]: I copied this example from the same file TextFieldBuilder.swift, almost every func there has this type of padding

**Comment by defraimo** [comment]
I fixed everything in the comments except the UIVIew padding thing, waiting for an answer

**Review by menesh2 — APPROVED** [review]

**Pull request merged**